plugins/modules/cloud/scaleway/scaleway_container.py

```diff
@@ -0,0 +1,417 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Scaleway Serverless container management module
+#
+# Copyright (c) 2022, Guillaume MARTINEZ <lunik@tiwabbit.fr>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: scaleway_container
+short_description: Scaleway Container management module
+author: Guillaume MARTINEZ (@Lunik)
+description:
+  - This module manages container on Scaleway account
+    U(https://developer.scaleway.com)
+extends_documentation_fragment:
+  - community.general.scaleway
+  - community.general.scaleway_waitable_resource
+
+
+options:
+  state:
+    type: str
+    description:
+      - Indicate desired state of the container.
+    default: present
+    choices:
+      - present
+      - absent
+
+  namespace_id:
+    type: str
+    description:
+      - Container namespace identifier.
+    required: true
+
+  region:
+    type: str
+    description:
+      - Scaleway region to use (for example fr-par).
```

Comment:
```suggestion
      - Scaleway region to use (for example C(fr-par)).
```
See https://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#linking-and-other-format-macros-within-module-documentation for how to format things.

Reply:
This formating issue has been corrected on all modules description